pkgs/development/python-modules/connexion/default.nix

```diff
@@ -1,70 +1,98 @@
 { lib
-, aiohttp
-, aiohttp-jinja2
-, aiohttp-remotes
-, aiohttp-swagger
+, fetchFromGitHub
 , buildPythonPackage
+, pythonOlder
+
+# build-system
+, poetry-core
+
+# dependencies
+, asgiref
 , clickclick
-, decorator
-, fetchFromGitHub
-, flask
+, httpx
 , inflection
 , jsonschema
-, openapi-spec-validator
-, packaging
-, pytest-aiohttp
-, pytestCheckHook
-, pythonOlder
+, jinja2
+, python-multipart
 , pyyaml
 , requests
-, swagger-ui-bundle
+, starlette
+, typing-extensions
+, werkzeug
+
+# optional-dependencies
+, a2wsgi
+, flask
+, py-swagger-ui
+, uvicorn
+
+# tests
+, importlib-metadata
+, pytest-aiohttp
+, pytestCheckHook
 , testfixtures
 }:
 
 buildPythonPackage rec {
   pname = "connexion";
-  version = "2.14.2";
-  format = "setuptools";
+  version = "3.0.0a7";
```

Comment:
seems that there is now a stable 3.0 release

Reply:
Updated, thanks!